### ISSUE TITLE
note@accid interaction with note@staff

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#488]
This used to display correctly, but some recent updates to verovio has broken it, or most likely`accid` elements have never been checked for cross-staff behavior:

<img width="833" alt="screen shot 2017-02-26 at 11 42 36 pm" src="https://cloud.githubusercontent.com/assets/3487289/23352635/46a5a052-fc7d-11e6-953a-b4f33999ea01.png">

When a note has a `@staff` value other than the home staff, the note is moved to the desired staff, but the accidental is displayed on the home staff (but uses the correct clef of the away staff).  Probably the `accid` element should check its parent `note` element for `@staff`.

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2017-02-26T23:43:32" version="0.9.14-dev-809d327">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-0000002018258896">
                        <staffGrp xml:id="staffgrp-0000000826935947" symbol="brace" barthru="true">
                            <staffDef xml:id="staffdef-0000001945781492" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4" n="1" lines="5" />
                            <staffDef xml:id="staffdef-0000000215969950" clef.shape="F" clef.line="4" meter.count="4" meter.unit="4" n="2" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000000172031887">
                        <measure xml:id="measure-L4" n="1">
                            <staff xml:id="staff-L4F2N1" n="1">
                                <layer xml:id="layer-L4F2N1" n="1">
                                    <note xml:id="note-L5F2" dur="4" oct="4" pname="e">
                                        <accid xml:id="accid-L5F2" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L6F2" dur="4" oct="4" pname="c" stem.dir="up">
                                        <accid xml:id="accid-L6F2" accid="s" />
                                    </note>
                                    <note xml:id="note-L7F2" dur="4" staff="2" oct="3" pname="b" stem.dir="up">
                                        <accid xml:id="accid-L7F2" accid="f" />
                                    </note>
                                    <note xml:id="note-L8F2" dur="4" staff="2" oct="3" pname="a" stem.dir="up">
                                        <accid xml:id="accid-L8F2" accid="f" />
                                    </note>
                                </layer>
                            </staff>
                            <staff xml:id="staff-L4F1N1" n="2">
                                <layer xml:id="layer-L4F1N1" n="1">
                                    <mRest xml:id="mrest-0000001608135043" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L9" right="end">
                            <staff xml:id="staff-L9F2N1" n="1">
                                <layer xml:id="layer-L9F2N1" n="1">
                                    <mRest xml:id="mrest-0000001292610832" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-L9F1N1" n="2">
                                <layer xml:id="layer-L9F1N1" n="1">
                                    <note xml:id="note-L10F1" dur="4" oct="3" pname="a">
                                        <accid xml:id="accid-L10F1" accid="f" />
                                    </note>
                                    <note xml:id="note-L11F1" dur="4" staff="1" oct="4" pname="c" stem.dir="down">
                                        <accid xml:id="accid-L11F1" accid="s" />
                                    </note>
                                    <note xml:id="note-L12F1" dur="4" staff="1" oct="4" pname="e" stem.dir="down">
                                        <accid xml:id="accid-L12F1" accid="f" />
                                    </note>
                                    <note xml:id="note-L13F1" dur="4" staff="1" oct="4" pname="f" stem.dir="down">
                                        <accid xml:id="accid-L13F1" accid.ges="n" />
                                    </note>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
Yes, this will be fixed in the next round of alignment refactoring.

**Comment by lpugin** [comment]
Fixed with 851d57f25d99362bf27378b1be57d8fe03f65d3e
